src/components/Category.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+function Category(Category) {
+  return (
+    <div
+      className="rounded-lg overflow-hidden shadow-2xl w-16 h-12 bg-gray-200"
+      id="card"
+    >
+      <div className="img-container" id="container">
+        <img
+          src={Category.data.Image}
+          alt="category-icon"
+          className="m-auto w-2/5"
+        />
+        <div className="text-xs mb-1 text-center text-yellow-500" id="title">
+          {Category.data.Title}
+        </div>
+      </div>
+    </div>
+  );
+}
+Category.defaultProps = {
+  data: {
+    Image: "https://image.flaticon.com/icons/png/512/1548/1548784.png",
+    Title: "Not Found",
+    id: "0",
+  },
```

Comment:
why are you nesting your object inside another `data` object, you can put your data directly 
```suggestion
    Image: "https://image.flaticon.com/icons/png/512/1548/1548784.png",
    Title: "Not Found",
    id: "0",
```

Reply:
i think its cleaner and i thought that was how its supposed to be

Reply:
It isn't really cleaner since you are just nesting an object inside another one, and since the outer object only has one field, then why bothering nesting?